toolkit/tools/imagegen/configuration/systemconfig_test.go

```diff
@@ -119,11 +120,11 @@ func TestShouldFailParsingInvalidKernelForRootfs_SystemConfig(t *testing.T) {
 
 	err := rootfsInvalidKernelConfig.IsValid()
 	assert.Error(t, err)
-	assert.Equal(t, "empty kernel entry found in the [KernelOptions] field (extra); remember that kernels are FORBIDDEN from appearing in any of the [PackageLists]", err.Error())
+	assert.Equal(t, "empty kernel entry found in the [KernelOptions] field (extra); remember that kernels are FORBIDDEN from appearing in any of the [PackageLists] or [Packages]", err.Error())
 
 	err = remarshalJSON(rootfsInvalidKernelConfig, &checkedSystemConfig)
 	assert.Error(t, err)
-	assert.Equal(t, "failed to parse [SystemConfig]: empty kernel entry found in the [KernelOptions] field (extra); remember that kernels are FORBIDDEN from appearing in any of the [PackageLists]", err.Error())
+	assert.Equal(t, "failed to parse [SystemConfig]: empty kernel entry found in the [KernelOptions] field (extra); remember that kernels are FORBIDDEN from appearing in any of the [PackageLists] or [Packages]", err.Error())
 }
 
 func TestShouldFailParsingBadKernelCommandLine_SystemConfig(t *testing.T) {
```

Comment:
Could you also add a short test for passing scenarios:
- use both `PackageLists` and `Packages`;
- use only one of each.

Reply:
I've added a passing scenario test for either packageList or Packages being empty. 

The test for both being used should be addressed in initial test of the valid system config as the default config uses both PackageLists and Packages.